### PR TITLE
add partest command and associated sbt magic

### PR DESCRIPTION
**Pull request by hrhino**
it's not as great as the scalac one, but I didn't look at that so I don't know exactly what.

### CODE REVIEW

File: .idea/hydra.xml [hunk 1]
```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="HydraSettings">
```
Comment (alexknvl) [inline .idea/hydra.xml]: Damn you, IJ, I didn't even notice it adding this file :rage:

File: build.sbt [hunk 2]
```diff
@@ -48,6 +33,19 @@ lazy val example = crossProject.module
...
+  .settings(
+    partestFramework,
+    libraryDependencies ++= List(
+      scalaOrganization.value % "scala-reflect"  % scalaVersion.value % Provided,
+      scalaOrganization.value % "scala-compiler" % scalaVersion.value % Provided,
+      partestDependency(scalaVersion.value) % Test,
+    ),
+  )
+
+commands in Global += Commandz.partestCommand(baseDirectory.value)
```
Comment (alexknvl) [inline build.sbt]: Could you write a comment explaining how to run tests? How to run pending, pos, neg, and individual tests.

Comment (hrhino) [reply build.sbt]: Yeah sure

File: .idea/hydra.xml (hunk 1 above)
Comment (NeQuissimus) [reply .idea/hydra.xml]: @alexknvl You owe each of us a Hydra license :P

**Comment by hrhino** [comment]
travis is busted but [it passes](https://travis-ci.org/scalaz/scalaz-plugin/builds/398057672)

**Pull request merged**